src/test/java/edu/ucsb/cs156/example/controllers/UCSBSubjectsControllerTests.java

```diff
@@ -48,9 +48,9 @@
     UCSBSubjectsService ucsbSubjectsService;
 
     @Test
-    public void api_UCSBSubjects_all__logged_out__returns_403() throws Exception {
+    public void api_UCSBSubjects_all__logged_out__returns_200() throws Exception {
         mockMvc.perform(get("/api/UCSBSubjects/all"))
-                .andExpect(status().is(403));
+                .andExpect(status().isOk());
```

Comment:
Wasn't this change already addressed in #72? Are we getting rid of the old PR?

Reply:
yes we are. Because it is quite literally 4 lines of change decided to have only 1 PR instead of 2

Reply:
Close #72